docs/src/main/sphinx/sql/delete.md

```diff
@@ -3,7 +3,7 @@
 ## Synopsis
 
 ```text
-DELETE FROM table_name [ WHERE condition ]
+DELETE FROM table_name [ @ branch_name ] [ WHERE condition ]
```

Comment:
@ebyhr, an explainer doc shows the @ sign always used within single quotes. Is that not necessary?

```
[`@` branch-name ]
```